testdata/well_known/transform_well_known.py

```diff
@@ -0,0 +1,74 @@
+import segyio
```

Comment:
Because all functions here rely on well_known.segy being created. Data is read from it and some modifications are made before final vds file creation.